src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1898,12 +1934,35 @@ def load_data(self):
                 else:
                     # this load in memmory
                     ext_data = np.array(ext_data_)
-                self.data[ext_data_name] = ext_data
+
+        if ext_data is None:
+            warnings.warn(f"Found no data for {self.extension_name}, extension should be re-computed.")
+
+        if keep:
+            self.data[ext_data_name] = ext_data
+
+    def _check_data_loadable(self):
```

Comment:
After discussing with @samuelgarcia , we agree this is an overkill. It's enough for the purpose of this PR to add `run_info` with run time and run_completed. Can you remove this @jonahpearl ?

Reply:
All set